pom.xml

```diff
@@ -1684,7 +1684,6 @@
                   -Xep:CheckReturnValue:OFF \
                   -Xep:MustBeClosedChecker:OFF \
                   -Xep:ReturnValueIgnored:OFF \
-                  -Xep:FutureReturnValueIgnored:ERROR \
```

Comment:
Except that it's default severity is [WARNING](https://errorprone.info/bugpattern/FutureReturnValueIgnored) and all warnings are disabled in the [configuration](https://github.com/apache/accumulo/blob/main/pom.xml#L1682)

Reply:
@dlmarion, sorry, my mistake. I thought it was under the Error pattern list vs the Warning pattern. I'll close this as not implemented. In a future PR related to some other EP related changes, I may consider moving the line beneath the 'warning' comment so keep separate Error patterns from Warning patterns. Would you have any objections to that re-arrangement?

Reply:
no, I have no issue with that. I would like to keep this at ERROR though, unless we remove the disable all warnings parameter, in case the issue is introduced in new code.

Reply:
No problem! Will definitely keep it as an error.